Add a build script for vector_add (#123)
nlesc-recruit/cudawrappers: README.dev.md

> ## Running the tests

Reviewer: ```suggestion
## Running the tests on DAS
```

Enter the `build` directory and run `make test`.

> If you are running the tests on DAS, you can send create a job with command using `srun`.

Reviewer: ```suggestion
If you are running the tests on DAS, you can run a job using `srun` command.
```

For instance,

```shell
srun -N 1 -C TitanX --gres=gpu:1 make test
```

> 

Reviewer: This command will run the tests in one of the worker nodes with a GPU device.

> 

Reviewer: ```suggestion
This command will run the tests in one of the worker nodes with a GPU device.
```
